### ISSUE TITLE
Feature request: Choose download source for Watch Later

### ISSUE THREAD

**Issue by crhatfield** [mikew/ss-plex.bundle#34]
Would be nice if I could be able to click on one of the downloading episodes and be able to choose a specific and known reliable/fast source instead of the Try Next Source option. Alternatively, if we could prioritize known sources based on our own download speeds or even if the program would store past known download speeds averages and base upon that even, that would be great too. Would also be great if the list of known sources would show file size and possibly past download speed averages so we could make a good judgement as to which source/file to download, for instance; if there are 3 from a specific source, we could choose the largest file (hopefully highest quality video, etc..)

**Comment by mikew** [comment]
Yeah this is desperately needed. Either a list of "preferred sources" (not sure how to work this in Plex) or adding the "Watch Now / Watch Later" buttons to the items in View All Sources

## 

Mike Wyatt
+1 (709) 749-3575
